### DISCUSSION TITLE
What must a path format look like on a MAC that I have to enter in Folder with Files

### DISCUSSION THREAD

**Discussion by MexxKrementor** [FlowiseAI/Flowise#1173] [Q&A]
I am using Flowise online. I would like to upload several documents with the node "Folder with Files". I have a Apple MAC and need to enter the folder path. What must the folder path look like that I have to enter so that the path is also recognized?

If I simply copy the folder path and enter it in the node, it does not work.

The path is:

/Volumes/Macintosh HD - Data/Downloads/Upload

Can you tell me from the example what I need to enter?

Thanks in advance!

**Comment by HenryHengZJ** [comment]
If you have FW deployed on cloud services like Render, Railway etc, the node wouldnt work. We are working on allowing users to get files from GoogelDrive, Dropbox

**Reply by xperttechnologies28** [reply]
Hi, I just wanted to know how I can read a directory of all the files in that directory. Do you have a Folder Loader as I don't seem to find it.

**Reply by gitwittidbit** [reply]
> If you have FW deployed on cloud services like Render, Railway etc, the node wouldnt work. We are working on allowing users to get files from GoogelDrive, Dropbox

How about Nextcloud or something else more attuned to data privacy?
